docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -707,6 +707,26 @@ The output of the query has the following history columns:
     - ``boolean``
     - Whether or not the operation appended data
 
+``$properties`` table
+~~~~~~~~~~~~~~~~~~~~~
+
+The ``$properties`` table provides access to general information about Delta
```

Comment:
I'd recommend keeping things simple and mention that the table gives access to the properties names and values of the Delta Lake table. See https://trino.io/docs/current/connector/hive.html#metadata-tables

Reply:
I've changed the wording slightly to distinguish between table configuration (minReaderVersion, etc.), table features (items of the features field of protocol entries), and table properties (custom properties). Arguably this is an implementation detail of Delta. Open to other ways to phrase this.